backend/controllers/Events.js

```diff
@@ -15,12 +15,25 @@ eventRouter.use(bodyParser.urlencoded({
 eventRouter.get('/list', function(req, res) {
   Event.find(function(err, events) {
     if (err) {
-      res.status(500).json(err);
+      return res.status(500).json(err);
     }
     var mappedEvents = events.map(q => q.toJSONFor());
     res.json(mappedEvents);
   });
-})
+});
+
+eventRouter.get('/event', function(req, res) {
+  var eventToGet = req.query;
```

Comment:
Instead of taking the whole query, it would be more readable to define the parameter we are passing in through the url.
```JavaScript
app.get('/id/:id', function (req, res) {
  const id = req.params.id;
  // ...
})
```

Reply:
Fixed the url!